reading-pool.md

```diff
@@ -33,7 +33,7 @@ Tech Worker Co-ops:
 
 Civics, Tech, Politics:
 
-- 🌐 [Paul Frazee "Information Civics"](https://infocivics.com/)
+- 🌐 Paul Frazee "Information Civics" ([Wayback Machine](https://web.archive.org/web/20201025044716/https://infocivics.com/), [Github](https://github.com/pfrazee/infocivics))
```

Comment:
Site no longer resolves. Found on Way Back Machine. 
Also the last iteration of the site redirected to GitHub making me think its not coming back.